catalog/video/catalog-mpegts-transformer/src/main/java/com/connexta/transformer/video/ListOfHandler.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import java.io.Serializable;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+import org.codice.ddf.libs.klv.KlvDataElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Attribute;
+
+/**
+ * This handler expects a KlvDataElement that matches the Class passed into
+ * {@link #ListOfHandler(String, Class)} generates a list of values returned by
+ * {@link KlvDataElement#getValue()}.
+ */
+public class ListOfHandler<T extends Serializable> extends BaseKlvHandler {
```

Comment:
There's probably a better name for this class. Maybe `ListOfBasicKlvDataTypesHandler`?